Cylenium.Tests/DriverTests.cs

```diff
@@ -39,6 +40,30 @@ public void Set_and_get_the_window_size()
             Assert.AreEqual(cy.WindowSize(), expectedSize);
         }
 
+        [Test]
+        [Category("Driver")]
+        public void Navigate_backward_and_forward()
+        {
+            var wait = cy.Wait();
+
+            cy.Visit("https://google.com");
+            wait.Until(_ => cy.Get("[name='q']"));
+            var first = cy.Title();
+
+            cy.Visit("https://www.ultimateqa.com");
+            wait.Until(_ => cy.Get("[href='https://ultimateqa.com/video-tutorials/']"));
+            var second = cy.Title();
+
+            cy.Go("backward", 1);
+            var last = cy.Title();
+            Assert.AreEqual(first, last, "Expected to be on " + first + " went backward to" + last);
```

Comment:
I wouldn't hold the variable `last` because the name is confusing as you read it. Technically, this is the current title you are on which you are testing against the first_title.

```python
Assert.AreEqual(first_title, cy.Title());
```